lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -207,11 +216,13 @@ namespace Js
 
         DynamicObject * functionKindObj = library->CreateObject();
 
-#define VALUE(ClassName, methodName, isStatic) library->AddMember(functionKindObj, PropertyIds::ClassName##_##methodName, JavascriptNumber::ToVar((int)FunctionKind::ClassName##_##methodName, scriptContext));
+#define VALUE(ClassName, methodName, propertyType) library->AddMember(functionKindObj, PropertyIds::ClassName##_##methodName, JavascriptNumber::ToVar((int)FunctionKind::ClassName##_##methodName, scriptContext));
         FUNCTIONKIND_VALUES(VALUE)
 #undef VALUE
 
         library->AddMember(builtInNativeInterfaces, PropertyIds::FunctionKind, functionKindObj);
+        library->AddMember(builtInNativeInterfaces, PropertyIds::POSITIVE_INFINITY, library->GetPositiveInfinite());
```

Comment:
small nit: update line 210 to account for these two new hard-coded properties

Reply:
good catch, I'll update that